### ISSUE TITLE
Ansible-review can't parse roles with dependices from custom modules

### ISSUE THREAD

**Issue by strangeman** [willthames/ansible-review#16]
Hi!

In my playbook, I use role `spring-boot`, which depends from role `maven-download`, which provides module `maven_download`.

Ansible-review fails with error:
```
Couldn't parse task at roles/spring-boot/tasks/extract_artifact.yml:17 (no action detected in task. This often indicates a misspelled module name, or incorrect module path.)
{ 'maven_download': 'group_id={{ item.group_id }} artifact_id={{ item.artifact_id }} version={{ item.artifact_version }} classifier=apidoc dest={{ doc_dest }}/{{ item.artifact_id }}.zip repository_url={{ maven_repo }} username={{ maven_repo_username }} password={{ maven_repo_password }}',
  'name': 'download {{ item.artifact_id }} java package',
  'register': 'download_result'}
```

**Comment by willthames** [comment]
Yeah, that's a difficult one. I'm really a little unsure as to better do this.

Currently the pragmatic way is to add module directories to the `ANSIBLE_LIBRARY` path. I know that's a rubbish solution, but (a) I believe that dependent roles is a recipe for disaster and (b) even if I didn't believe that, there's no code in ansible-review that does the dependency analysis, because it's hard, and because ansible-review is a file-based review tool (it can find a meta/main.yml so what you're asking isn't impossible, just unlikely in the absence of a useful PR)

**Comment by strangeman** [comment]
>add module directories to the ANSIBLE_LIBRARY path
Yeah, this workaround is acceptable for me. We use only 3 roles which provide custom modules, so it will be not hard.

BTW, ansible-lint and ansible-review are awesome tools, thanks a lot!

**Comment by ssbarnea** [comment]
Any plans to fix this? At least if we would have a config option inside `.ansible-lint` config file, we could use this one to avoid adding an extra environment variable which is extremely problematic for using the linter from other tools where ability to define environment variable may be very limited (gui apps/editors). The linter should be usable and controllable without having to hack environment variables.

**Comment by willthames** [comment]
PRs accepted @ssbarnea

**Comment by MaciejKucia** [comment]
Taking _ansible.cfg_ into consideration would be fantastic.
